### PR TITLE
Update peruvian channels

### PR DESCRIPTION
**Pull request by sguinetti**
From #51

This PR add cities and websites for entries. Also update website for HISPAM channels.

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -28929,4 +28930,4 @@ Zvezda.ru,Zvezda,,,RU,,,c/RU,rus,general,FALSE,,,,https://i.imgur.com/n9zLVyR.pn
 ZvezdaPlus.ru,Zvezda Plus,,,RU,,,c/RU,rus,,FALSE,,,https://tvzvezda.ru/sp/zvezda-plus/,https://cdn.red-media.ru/wp-content/uploads/2021/02/Star_PLUS.png
 ZvezdaTV.rs,Zvezda TV,,,RS,,,c/RS,srp,,FALSE,,,,https://mts.rs/oec/images/tv_channels/8cff6095819ea749b6580f73c7d31d26.jpg
 ZVTAVS.nl,ZVT-AVS,,,NL,,,c/NL,nld,,FALSE,,,,https://i.imgur.com/9PG10sW.png
-Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png
\ No newline at end of file
+Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png
```
Comment (freearhey) [inline data/channels.csv:28933]:
```suggestion
Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]
The file must not contain empty lines.

File: data/channels.csv (hunk 1 above)
Comment (sguinetti) [reply data/channels.csv:28933]: The text editor does not allow me to save without spaces. So it's a problem unrelated to mine. Possibly, I will try to correct it in the future.

Comment (sguinetti) [reply data/channels.csv:28933]: I have tried the website, but it won't let me save without the spaces in your suggestion.

**Review by freearhey — CHANGES_REQUESTED** [review]
I removed the blank line, but there are still 284 errors in the file. You can see the full list with the command:

```
npm run db:validate -- data/channels.csv 
```

**Comment by sguinetti** [comment]
Ready for review

**Comment by sguinetti** [comment]
Finally, ready for review.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Excellent work!

**Pull request merged**